packages/zoe/src/types.js

```diff
@@ -349,17 +347,86 @@
  * @property {Issuer} issuer
  * @property {AmountMath} amountMath
  *
- * @typedef {Object} InstallationRecord
- * @property {InstallationHandle} handle - opaque identifier for the installation
- * @property {SourceBundle} bundle - contract code
- *
- * @typedef {Object} OfferStatus
- * @property {OfferHandle[]} active
- * @property {OfferHandle[]} inactive
- *
- * @typedef {(offerHandle: OfferHandle) => void} OfferHook
+ * @typedef {Record<Keyword,Amount>} Allocation
```

Comment:
```suggestion
 * @typedef {AmountKeywordRecord} Allocation
```
makes is explicit that they are the same structural type, but `Allocation` names a more specific intention.